doc/en/user/source/extensions/geofence-server/installing.rst

```diff
@@ -3,18 +3,83 @@
 Installing the GeoServer GeoFence Server extension
 ==================================================
 
-#. Visit the :website:`website download <download>` page, locate your release, and download: :download_extension:`geofence-server`
+
+Install the plugin
+------------------
+
+ #. Visit the :website:`website download <download>` page, locate your release, and download: :download_extension:`geofence-server`
    
-   The download link will be in the :guilabel:`Extensions` section under :guilabel:`Other`.
+    The download link will be in the :guilabel:`Extensions` section under :guilabel:`Other`.
    
-   .. warning:: Ensure to match plugin (example |release| above) version to the version of the GeoServer instance.
+    Make sure to match the plugin version (e.g. |release| above) to the version of the GeoServer instance.
 
  #. Extract the files in this archive to the :file:`WEB-INF/lib` directory of your GeoServer installation.
+ 
+    .. warning:: this plugin will install a version of the `H2 <http://www.h2database.com>`__  library that **is not compatible** 
```

Comment:
I would indicate that it's also safe to remove (it is, right?), in other words, that it's possible to use GeoFence with NetCDF/Grib provided the H2 v2 database is removed and the rule storage is moved to an external database.

I see it's written down in the same page, but some careless user might not read to the end, better to inform them now and indicate there are instructions later in the page.